roles/st2chatops/tasks/main.yml

```diff
@@ -0,0 +1,71 @@
+---
+- name: Install latest st2chatops package
+  become: yes
+  package:
+    name: st2chatops
+    state: latest
+  when: st2chatops_version == "latest"
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Install pinned st2chatops package
+  become: yes
+  package:
+    name: st2chatops={{ st2chatops_version }}
+    state: present
+  when: st2chatops_version != "latest"
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Add st2_api_key in st2chatops.env
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '(?<=ST2_API_KEY=)(.*)$'
+    replace: '{{ st2chatops_ST2_API_KEY }}'
+  when: st2chatops_ST2_API_KEY != "CHANGE-ME-PLEASE"
+  register: st2chatops_api_key
+  tags: st2chatops
+
+- name: Comment Username, Password and Auth URL, if API_KEY provided
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '{{ item }}'
+    replace: '# \1'
+  with_items:
+    - '^(export ST2_AUTH_URL.*)'
+    - '^(export ST2_AUTH_USERNAME.*)'
+    - '^(export ST2_AUTH_PASSWORD.*)'
+  when: st2chatops_api_key.changed
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Uncomment Hubot "{{ st2chatops_HUBOT_ADAPTER|upper }}" adapter in "/opt/stackstorm/chatops/st2chatops.env"
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '^# (export HUBOT_ADAPTER={{ st2chatops_HUBOT_ADAPTER }})$'
+    replace: '\1'
+  when: st2chatops_api_key.changed and st2chatops_HUBOT_ADAPTER in supported_hubot_adapter
```

Comment:
`st2chatops_HUBOT_ADAPTER in supported_hubot_adapter`

In this case it will skip the task if adapater is wrong.
I think it makes sense to move this validation logic in separated task and FAIL playbook execution early and loud if user provided invalid adapter.